src/LatentGPs.jl

```diff
@@ -1,5 +1,28 @@
 module LatentGPs
 
-greet() = print("Hello World!")
+using Distributions
+using KernelFunctions
+using AbstractGPs
+using LinearAlgebra
+using Random
+using Requires
+
+import Statistics
+import Distributions
+
+export LatentGP
+
+export logpdf, rand, trainable, randf, randy
+
+export GaussianLikelihood, PoissonLikelihood
+
+
+include("latent_gp.jl")
+include("likelihoods/likelihoods.jl")
+
+function __init__()
+    @require Flux="587475ba-b771-5e3f-ad9e-33799f191a9c" include("trainable.jl")
+end
```

Comment:
I don't think this is needed anymore (not in KernelFunctions either). The logic was separated into https://github.com/FluxML/Functors.jl. I.e., I guess it would be sufficient to depend on Functors (which should be a light dependency we can take I guess?), and just add
```julia
@functor GaussianLikelihood
```
after the definition of `GaussianLikelihood`.

Reply:
Would be nice to get rid of Requires since it can increase loading times quite a bit, and it would be more generally applicable and not only be restricted to training with Flux.